src/site/xdoc/index.xml

```diff
@@ -27,68 +27,59 @@ under the License.
-</li>
-    <li>Thrift gateway and a REST-ful Web service that supports XML, Protobuf, and binary data encoding options
-</li>
-    <li>Extensible jruby-based (JIRB) shell
-</li>
-    <li>Support for exporting metrics via the Hadoop metrics subsystem to files or Ganglia; or via JMX
-</li>
-</ul>
-</p>
-</section>
-     <section name="More Info">
-   <p>See the <a href="http://hbase.apache.org/book.html#arch.overview">Architecture Overview</a>, the <a href="http://hbase.apache.org/book.html#faq">Apache HBase Reference Guide FAQ</a>,
-    and the other documentation links.
-   </p>
-   <dl>
-     <dt>Export Control</dt>
-   <dd><p>The HBase distribution includes cryptographic software. See the export control notice <a href="export_control.html">here</a>
-   </p></dd>
-     <dt>Code Of Conduct</dt>
-   <dd><p>We expect participants in discussions on the HBase project mailing lists, Slack and IRC channels, and JIRA issues to abide by the Apache Software Foundation's <a href="http://apache.org/foundation/policies/conduct.html">Code of Conduct</a>. More information can be found <a href="coc.html">here</a>.
-   </p></dd>
- </dl>
-</section>
+      <p>
+        <ul>
+          <li>Linear and modular scalability.</li>
+          <li>Strictly consistent reads and writes.</li>
+          <li>Automatic and configurable sharding of tables</li>
+          <li>Automatic failover support between RegionServers.</li>
+          <li>Convenient base classes for backing Hadoop MapReduce jobs with Apache HBase tables.</li>
+          <li>Easy to use Java API for client access.</li>
+          <li>Block cache and Bloom Filters for real-time queries.</li>
+          <li>Query predicate push down via server side Filters</li>
+          <li>Thrift gateway and a REST-ful Web service that supports XML, Protobuf, and binary data encoding options</li>
+          <li>Extensible jruby-based (JIRB) shell</li>
+          <li>Support for exporting metrics via the Hadoop metrics subsystem to files or Ganglia; or via JMX</li>
+        </ul>
+      </p>
+    </section>
+    <section name="More Info">
+      <p>See the <a href="book.html#arch.overview">Architecture Overview</a>, the <a href="book.html#faq">Apache HBase Reference Guide FAQ</a>, and the other documentation links.</p>
+      <dl>
+        <dt>Export Control</dt>
+        <dd><p>The HBase distribution includes cryptographic software. See the export control notice <a href="export_control.html">here</a></p></dd>
+        <dt>Code Of Conduct</dt>
+        <dd><p>We expect participants in discussions on the HBase project mailing lists, Slack and IRC channels, and JIRA issues to abide by the Apache Software Foundation's <a href="http://apache.org/foundation/policies/conduct.html">Code of Conduct</a>. More information can be found <a href="coc.html">here</a>.</p></dd>
+        <dt>License</dt>
+        <dd><p>Apache HBase is licensed under the <a href="licenses.html">Apache License, Version 2.0</a></p></dd>
```

Comment:
The `licenses.html` file is linked under Apache HBase Project > License in the header. Whimsy checks for `^https?://.*apache.org/licenses/?$` so it might be better to modify licenses.html to the apache.org link

Reply:
You are right, I changed it. I also checked how [zookeeper](https://zookeeper.apache.org/) does, their [whimsy board](https://whimsy.apache.org/site/project/zookeeper) looks OK for the license check.